lib/floe/workflow/context.rb

```diff
@@ -67,7 +67,7 @@ def status
           "pending"
         elsif running?
           "running"
-        elsif state["Error"]
+        elsif output&.[]("Error")
```

Comment:
@kbrock could we move this check to a `failure?` method like we have above for `started?` and `running?` ?